src/components/Amount.tsx

```diff
@@ -22,6 +22,7 @@ interface AmountProps {
 }
 
 const Amount: FC<AmountProps> = ({ value, className }) => {
+  console.log('value' + value?.toString())
```

Comment:
is it a debug leftover or you really want it?

Reply:
Nice catch!